plutus-core/plutus-core/src/PlutusCore/Builtin/Meaning.hs

```diff
@@ -109,8 +109,8 @@ throw an error instead of overflowing its first argument.
 One denotation from each builtin is grouped into a 'BuiltinSemanticsVariant'.
 Each Plutus Language version is linked to a specific 'BuiltinSemanticsVariant'
 (done by plutus-ledger-api); e.g. plutus-v1 and plutus-v2 are linked to
-'DefaultFunSemanticsVariant1', whereas plutus-v3 changes the set of denotations
-to 'DefaultFunSemanticsVariant2' (thus fixing 'ConsByteString').
+'DefaultFunSemanticsVariantA', whereas plutus-v3 changes the set of denotations
+to 'DefaultFunSemanticsVariantC' (thus fixing 'ConsByteString').
```

Comment:
The very first one should be `DefaultFunSemanticsVariant1` and `DefaultFunSemanticsVariantB` rather than just the former?